packages/core/src/runner/helper.ts

```diff
@@ -78,8 +75,8 @@ export function createPipeline(components: PipcookComponentResult[], self: Pipco
   const firstComponent = components[0];
   firstComponent.status = PipcookComponentResultStatus.Running;
   logCurrentExecution(firstComponent, logType);
-  const insertParams: InsertParams = {
-    pipelineId: self.pipelineId,
+  const insertParams = {
+    runId: self.runId,
```

Comment:
In our convention, it's `componentId`?

Reply:
This is the id of each run. For example, a pipeline id can corresponds to multiple run ids. Or we can call it job id